docs/pipeline.rst

```diff
@@ -110,52 +124,43 @@ Adding test runs
 ~~~~~~~~~~~~~~~~
 
 Test runs can be added by creating a new job in the `.gitlab-ci.yml file in the www-config repository`_
-with the desired variables. For example, if you wanted to run tests in Firefox on both Windows 10 and
-OS X, against our dev environment, you could create the following clauses:
+with the desired variables. For example, if you wanted to run the smoke tests in IE10, you could create the
+following clauses:
 
 .. code-block:: yaml
 
-  dev-test-firefox-osx:
-    extends:
-      - .dev
-      - .test
+  .ie10:
     variables:
-      BROWSER_NAME: firefox
-      BROWSER_VERSION: latest
-      PLATFORM: OS X 10.11
+      BROWSER_NAME: internet explorer
+      BROWSER_VERSION: "10.0"
+      PLATFORM: Windows 8
+      MARK_EXPRESSION: smoke
 
-  dev-test-firefox-win10:
+  test-ie10-saucelabs:
     extends:
-      - .dev
       - .test
-    variables:
-      BROWSER_NAME: firefox
-      BROWSER_VERSION: latest
-      PLATFORM: Windows 10
+      - .ie10
+      - .saucelabs
 
 You can use `Sauce Labs platform configurator`_ to help with the parameter values.
 
+Pushing to the integration tests branch
```

Comment:
The `run-intergration-tests` branch is not setup yet b/c it relied on the -now defunct- `oregon-b` cluster.

@pmac what do we need to do to move this to GCP?

Reply:
I filed https://github.com/mozilla/bedrock/issues/9598 to fix the run-integration-tests branch